src/azure-map/atlas-map/atlas-map.component.ts

```diff
@@ -88,23 +88,29 @@ export class AtlasMapComponent implements OnInit, AfterContentInit, OnChanges {
     this.popupView = this.popupsContainer.createEmbeddedView(this.popupTemplate, context);
   }
 
+  pinLayerOptions(item): PinLayerOptions {
```

Comment:
What I meant was that user should provide these configuration options, not hardcoding them in a module